docs/docs/getting-started/basic/data-formatting.md

```diff
@@ -23,3 +25,22 @@ but you can [add custom formats](/docs/getting-started/customizing-things/custom
 If your dataset is already formatted like this, great! Otherwise, this is the only bit where coding is required.
 You can either convert it yourself (via a python or bash script, whatever you're comfortable with), or you can register
 a [custom data reader](/docs/getting-started/customizing-things/custom-data-format/) to support your dataset format.
+
+
+## Datasets Organization
```

Comment:
"organization" here does not translate from Italian :D I'd say `Datasets Folder Structure`

Reply:
Organizing? Because it does not have to be a folder.

Reply:
Ok